front-end/src/App.tsx

```diff
@@ -50,9 +51,9 @@ function App() {
 
       // Keep track of a Node representing our own server
       setNodes([...nodes, {
-        "host": host,
-        "username": loggedInUser?.username, 
-        "password": loggedInUser?.password
+        host: host,
+        username: loggedInUser?.username, 
+        password: loggedInUser?.password
```

Comment:
Just refactoring this to create keys without using quotes, was too used to Python dictionaries.